packages/eslint-plugin/rules/array-bracket-spacing/array-bracket-spacing._js_.ts

```diff
@@ -188,14 +188,14 @@ export default createRule<RuleOptions, MessageIds>({
 
       const openingBracketMustBeSpaced
                 = firstElement && options.objectsInArraysException && isObjectType(firstElement)
-                || firstElement && options.arraysInArraysException && isArrayType(firstElement)
-                || options.singleElementException && node.elements.length === 1
+                  || firstElement && options.arraysInArraysException && isArrayType(firstElement)
+                  || options.singleElementException && node.elements.length === 1
                   ? !options.spaced : options.spaced
```

Comment:
looks like something breaks in `indent` here

Reply:
oh! I see, it is correct. The ternary expression's condition is the whole multi-line logical expression.